### PR TITLE
add transitions animations

### PR DESCRIPTION
**Pull request by dgore7**
transitions and animations

### CODE REVIEW

File: src/lib/components/ControlPad.svelte
Comment (oodtoon) [inline src/lib/components/ControlPad.svelte:1]: remove console.log

**Review by oodtoon — COMMENTED** [review]
Must remove console.log statements but other than that good job. You've come a long way and are learning so much.

File: src/lib/components/ControlPad.svelte [hunk 1]
```diff
@@ -84,24 +85,36 @@
...
-				on:click={() => handleBallClick(ball)}
-				disabled={ball.isDead || (isGameOver && !isDeadBallMode)}
-			>
-				<Ball {ball} size={'large'} />
-			</button>
-		{/if}
+	{#each game.currentRack.gameBalls as ball (ball.number)}
+		<div class="relative">
+			{#if ball.isDead}
+				<button
```
Comment (oodtoon) [inline src/lib/components/ControlPad.svelte:91]: <button
					class="relative"
					class:hidden={!ball.isDead}
					on:click={() => removeDeadBall(ball)}
					disabled={!isDeadBallMode}
					in:send|global={{ key: ball.number }}
					out:receive|global={{ key: ball.number }}
					on:transitionend={(e) => e.currentTarget.classList.remove('absolute')}
				>

**Review by oodtoon — APPROVED** [review]

File: src/lib/components/ControlPad.svelte [hunk 2]
```diff
@@ -84,24 +85,36 @@
...
+					class="relative"
+					class:hidden={!ball.isDead}
+					on:click={() => removeDeadBall(ball)}
+					disabled={!isDeadBallMode}
+					in:send|global={{ key: ball.number }}
+					out:receive|global={{ key: ball.number }}
+					on:transitionstart={console.log}
```
Comment (dgore7) [inline src/lib/components/ControlPad.svelte]:
```suggestion
```

**Pull request merged**